scripts/rollup/plugins/closure-plugin.js

```diff
@@ -10,7 +10,7 @@ function compile(flags) {
   return new Promise((resolve, reject) => {
     const closureCompiler = new ClosureCompiler(flags);
     closureCompiler.run(function(exitCode, stdOut, stdErr) {
-      if (!stdErr) {
+      if (!stdErr || exitCode === 0) {
```

Comment:
Should this be the only condition then? Why both?

Reply:
yeap. I will make the change.